lib/global_id/railtie.rb

```diff
@@ -14,14 +14,16 @@ class Railtie < Rails::Railtie # :nodoc:
     config.eager_load_namespaces << GlobalID
 
     initializer 'global_id' do |app|
+      default_expires_in = 1.month
+      default_app_name = app.railtie_name.remove('_application').dasherize
 
-      app.config.global_id.app ||= app.railtie_name.remove('_application').dasherize
-      GlobalID.app = app.config.global_id.app
-
-      app.config.global_id.expires_in ||= 1.month
-      SignedGlobalID.expires_in = app.config.global_id.expires_in
+      GlobalID.app = default_app_name
+      SignedGlobalID.expires_in = default_expires_in
```

Comment:
This now overrides config that was set outside an initializer, in `config/application.rb` and `config/environments/*.rb`. Think we'll need the assignment style you use in the `after_initialize` block.

Reply:
Thanks again for the review @jeremy! I've copied and pasted it for now since extracting it to a method and pass `app` feels kinda wrong. Let me know if you think this needs to be DRYed up.